portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,72 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 660px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#quote-container {
+  margin-top: 10px;
 }
+
+p {
+  font-size: 18px;
+  line-height: 22px;
+}
+
+figure {
+  display: inline-block;
+  text-align: center;
+}
+
+h1 {
+  text-align: center;
+  font-size: 72px;
+}
+
+h5 {
+    font-weight: normal;
+    font-size: 20px;
```

Comment:
2 space indention here?

Reply:
are you saying to make it two spaces or asking if it is two spaces? Right now it's one space in both lines.

Reply:
I mean it's currently 4 spaces, but to keep consistency with other places you should change it to 2 spaces.